docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -813,8 +813,8 @@ The main piece to configure is the enrich policy:
 | `type`             | yes       | -                    | The policy type.
 | `indices`          | yes       | -                    | The indices to fetch the data from.
 | `query`            | no        | `match_all` query    | The query to be used to select which documents are included.
-| `enrich_key`       | yes       | -                    | The field that the enrich processor will query against.
-| `enrich_values`    | yes       | -                    | The fields to include in the enrich index.
+| `match_field`      | yes       | -                    | The field that the enrich processor will query against.
+| `enrich_fields`    | yes       | -                    | The fields to include in the enrich index.
```

Comment:
How about : 
`The fields to include in the enrich index.`
to
`The fields that will be available to enrich the input document`